hieradata/class/monitoring.yaml

```diff
@@ -1,7 +1,5 @@
 ---
 
-govuk_cdnlogs::log_dir: '/var/log/cdn'
```

Comment:
Do we need to keep this so that it is used as input for the ensure => absent?

Reply:
I trust your knowledge of Puppet more than mine! Reinsated.